modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -52,6 +52,7 @@
             <class name="org.wso2.am.integration.tests.other.SameVersionAPITestCase" />
             <class name="org.wso2.am.integration.tests.other.APIM678ApplicationCreationTestCase" />
             <class name="org.wso2.am.integration.tests.other.APIMANAGER3226APINameWithDifferentCaseTestCase"/>
+            <class name="org.wso2.am.integration.tests.other.APIEndpointTypeUpdateTestCase"/>
```

Comment:
Let's commit the testng file at the end with all the tests included.

Reply:
removed